variants/feather_m0/debug_scripts/variant.gdb

```diff
@@ -1,31 +0,0 @@
-#
```

Comment:
We *definitely* want to keep this!  This is what we need for debugging with gdb and openocd. Please revert, and add similar for each of our boards.